src/js/utils/browser.ts

```diff
@@ -50,13 +50,18 @@ export const isAndroid = () => userAgentMatch(/Android/i) && !userAgentMatch(/Wi
 
 export const isMobile = () => isIOS() || isAndroid() || userAgentMatch(/Windows Phone/i);
 
-export function isIframe(): boolean {
+export const isIframe = function(): boolean {
+    if (typeof isIframe.mock_ === 'boolean') {
```

Comment:
Wasn't sure of a better way to do this. Ideally we would have a webpack plugin for the test build that allows us to stub functions and return values, but I couldn't find anything.